static/js/directives/report-image.js

```diff
@@ -0,0 +1,16 @@
+angular.module('inboxDirectives').directive('reportImage',
+  function(DB) {
+    return {
+      template: '<div class="loader"></div>',
+      link: function(scope, element, attr) {
+        DB().getAttachment(attr.report, attr.path, { binary:false })
```

Comment:
I don't think `binary:false` is needed. I can't see it anywhere on [PouchDB documentation](https://pouchdb.com/api.html#get_attachment) for this function.